scripts/ws-realm.sh

```
@@ -206,15 +284,143 @@ ws_classify_target_name() {
+# directory names, components declared in the merged ecosystem config, and
+# "<component>/<nested>" for a repo nested inside a component's working tree.
 # Exits non-zero with a kind-neutral message when nothing resolves.
+#
+# Every repo-touching verb funnels through here, so nested support is added once
+# and inherited rather than taught to each verb separately.
+WS_TARGET_NESTED_HOST=""
+WS_TARGET_NESTED_PATH=""
 ws_resolve_target() {
     local name="$1"
 
+    WS_TARGET_NESTED_HOST=""
```

Shellcheck (0.11.0)

[warning] WS_TARGET_NESTED_HOST appears unused. Verify use (or export if used externally).

(SC2034)